product_docs/docs/pgd/5.8/ddl/ddl-managing-with-pgd-replication.mdx

```diff
@@ -33,8 +33,7 @@ $ddl$);
 ```
 
 We recommend using the [`bdr.run_on_all_nodes()`](/pgd/5.8/reference/functions#bdrrun_on_all_nodes) technique with `CREATE
-INDEX CONCURRENTLY`, noting that DDL replication must be disabled for the whole
-session because `CREATE INDEX CONCURRENTLY` is a multi-transaction command.
+INDEX CONCURRENTLY`.
```

Comment:
We don't anymore in 6 because CIC is anyway executed concurrently now via replication